src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -182,7 +182,7 @@ def _apply_settings_file(self) -> None:
                     if self._params in (["settings", "--sample"], ["settings", "--gs"]):
                         return
                     msg = "Settings file cannot be empty."
-                    raise ValueError(msg)
+                    raise ValueError(msg)  # noqa: TRY301
```

Comment:
Are you sure that just using the simple raise from https://docs.astral.sh/ruff/rules/raise-within-try/ would break the code here?